internal/highway/http/crud_object_test.go

```diff
@@ -0,0 +1,58 @@
+package core_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	hw "github.com/sonr-io/sonr/internal/highway"
+	ot "github.com/sonr-io/sonr/x/object/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgCreateObject_ValidateBasic(t *testing.T) {
```

Comment:
This should be removed as it does depdends on the block chain to be running and to connect. will fail in a unit test enviorment.